app/models.py

```diff
@@ -8,6 +8,7 @@
 
 class Permission:
     GENERAL = 0x01
+    WORKER = 0x10
```

Comment:
Can we rename this to `AGENCYWORKER`? In case there are other types of workers which will be added in the future.